.github/workflows/helm-release.yaml

```diff
@@ -58,3 +58,16 @@ jobs:
           token: "${{ secrets.GITHUB_TOKEN }}"
           linting: off
           charts_dir: charts-tmp
+
+      - name: Login to GitHub Container Registry
+        run: |
+          helm registry login --username ${GITHUB_ACTOR} --password ${{ secrets.GITHUB_TOKEN }} ghcr.io
+      
+      - name: Publish OCI Charts
+        run: |
+          for dir in `find charts-tmp -maxdepth 1 -mindepth 1 -type d -print`; do
+            chart=${dir##*/}
+            echo "Found chart: ${chart}"
+            helm package charts-tmp/${chart} --destination dist
```

Comment:
Does this directory `dist` have to exist before running this command?

Reply:
no